### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by bmeike**
This change should not affect behavior in any way: it is simply code cleanup.

### CODE REVIEW

File: lib/src/shared/main/java/com/couchbase/lite/Dictionary.java [hunk 1]
```diff
@@ -98,6 +101,7 @@ public int count() {
      * @param key the key.
      * @return the object value or nil.
      */
+    @NonNull
```
Comment (Sandychuang8) [inline lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: inconsistent with line 102 NonNull vs return nil

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: Good call!  Fixing!

Comment (Sandychuang8) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: and I think it allows to return null..

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: I don't think so...  `MDict.get` cannot return `null`...

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: fixed

Comment (Sandychuang8) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: I checked .Net and looks like there is chance that getValue can return null.. please double check with @pasin ?

Comment (Sandychuang8) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: I thought getValue method returns object? not MValue..

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: MValue is an Object

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: Look at the code down there in MDict.java and assure yourself that this function cannot return null

Comment (pasin) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: The return value is nullable from the API spec. The MValue is an internal class and Java shouldn't return the MValue object directly. For example, when the key doesn't exist, the null value should be returned.

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: Certainly not relevant to this PR.  If you have found a bug, please file it.

Comment (bmeike) [reply lib/src/shared/main/java/com/couchbase/lite/Dictionary.java]: NVM..  I filed it: https://issues.couchbase.com/browse/CBL-542

**Pull request merged**